__tests__/utilities.test.ts

```diff
@@ -1,297 +1,300 @@
-var couponable = require('.');
```

Comment:
added `jest` instead of `mocha`, it seems that a `Makefile` is used to run the test via mocha, do we want to clean up the mocha references?